modules/fedimint-lnv2-common/src/gateway_api.rs

```diff
@@ -229,8 +229,8 @@ impl PaymentFee {
     /// This is the fee the gateway uses to cover transaction fees with the
     /// federation.
     pub const TRANSACTION_FEE_DEFAULT: PaymentFee = PaymentFee {
-        base: Amount::from_sats(50),
-        parts_per_million: 5_000,
+        base: Amount::from_sats(2),
+        parts_per_million: 100,
```

Comment:
@m1sterc001guy This might be a bit low actually. 

I did an analysis on failure rate increase a while ago (unfortunately the raw data is private) and putting in your values would lead to a 10% increase in failure rate with a rather well-connected node.

Based on the data I analyzed I'd recommend a 2-2.5 sat base fee and a 3000-3500ppm proportional fee. This leads to an increase between 0.03% and 0.1% in failures vs. the old 2sat+5000ppm fee, which is reasonable for the cost reduction imo.

Reply:
Changed to 3000 ppm